source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -72,221 +84,20 @@ Third party libraries are packaged into Python wheels and uploaded to PyPI (if p
-
-   .. tab-item:: Java/C++
-      :sync: javacpp
-
-      ``https://frcsdk.reduxrobotics.com/ReduxLib_2024.json``
-
-   .. tab-item:: Python
-      :sync: python
-
-      Not yet available
-
-[Playing With Fusion Driver](https://www.playingwithfusion.com/docview.php?docid=1205) - Library for all PWF devices including the Venom motor/controller
-
-.. tab-set::
-
-   .. tab-item:: Java/C++
-      :sync: javacpp
-
-      ``https://www.playingwithfusion.com/frc/playingwithfusion2024.json``
-
-   .. tab-item:: Python
-      :sync: python
-
-      Community-supported packages:
-
-      * PyPI package: ``robotpy[playingwithfusion]`` or ``robotpy-playingwithfusion``
-      * In ``pyproject.toml``: ``robotpy_extras = ["playingwithfusion"]``
-
-[Kauai Labs](https://pdocs.kauailabs.com/navx-mxp/software/roborio-libraries/) - Libraries for NavX-MXP, NavX-Micro, and Sensor Fusion
-
-.. tab-set::
-
-   .. tab-item:: Java/C++
-      :sync: javacpp
-
-      ``https://dev.studica.com/releases/2024/NavX.json``
-
-   .. tab-item:: Python
-      :sync: python
-
-      Community-supported packages:
-
-      * PyPI package: ``robotpy[navx]`` or ``robotpy-navx``
-      * In ``pyproject.toml``: ``robotpy_extras = ["navx"]``
-
-[REV Robotics REVLib](https://docs.revrobotics.com/brushless/spark-flex/revlib) - Library for all REV devices including SPARK Flex, SPARK MAX, and Color Sensor V3
-
-.. tab-set::
-
-   .. tab-item:: Java/C++
-      :sync: javacpp
-
-      ``https://software-metadata.revrobotics.com/REVLib-2024.json``
-
-   .. tab-item:: Python
-      :sync: python
-
-      Community-supported packages:
-
-      * PyPI package: ``robotpy[rev]`` or ``robotpy-rev``
-      * In ``pyproject.toml``: ``robotpy_extras = ["rev"]``
-
-### Community Libraries
-
-[PhotonVision](https://docs.photonvision.org/en/latest/docs/programming/photonlib/adding-vendordep.html) - Library for PhotonVision CV software
-
-.. tab-set::
-
-   .. tab-item:: Java/C++
-      :sync: javacpp
-
-      ``https://maven.photonvision.org/repository/internal/org/photonvision/photonlib-json/1.0/photonlib-json-1.0.json``
-
-   .. tab-item:: Python
-      :sync: python
-
-      * PyPI package: ``photonlibpy``
-      * In ``pyproject.toml``: ``requires = ["photonlibpy"]``
-
-[PathPlanner](https://pathplanner.dev/home.html) - Library for PathPlanner
-
-.. tab-set::
-
-   .. tab-item:: Java/C++
-      :sync: javacpp
-
-      ``https://3015rangerrobotics.github.io/pathplannerlib/PathplannerLib.json``
-
-   .. tab-item:: Python
-      :sync: python
-
-      * PyPI package: ``pathplannerlib``
-      * In ``pyproject.toml``: ``requires = ["pathplannerlib"]``
-
-[ChoreoLib](https://sleipnirgroup.github.io/Choreo/choreolib/installation/) - Library for reading and following trajectories generated by [Choreo](https://sleipnirgroup.github.io/Choreo/)
-
-.. tab-set::
-
-   .. tab-item:: Java/C++
-      :sync: javacpp
-
-      ``https://sleipnirgroup.github.io/ChoreoLib/dep/ChoreoLib.json``
+### How Does It Work?
```

Comment:
This feels like it can be tabs as well

Reply:
I disagree that How does it work should go in tabs.  It is critical for placing offline install files and really has nothing to do with the install.  All of the other tabs are direct ways to install a vendordep

Reply:
There's 3 sections for different languages that you don't need to read unless you're using that language. Isn't that what we use tabs for?

Reply:
I understand now you wanted to add a new set of tabs for it.  I thought you meant adding a tab to the one above it.

I will add the tab to it.